Solutions/Dynatrace/Playbooks/Enrich-DynatraceAppSecAttackMSDefender365/azuredeploy.json

```diff
@@ -0,0 +1,974 @@
+                                "schema": {
+                                    "properties": {
+                                        "AttackIdentifier": {
+                                            "items": {
+                                                "type": "string"
+                                            },
+                                            "type": "array"
+                                        },
+                                        "AttackState": {
+                                            "items": {
+                                                "type": "string"
+                                            },
+                                            "type": "array"
+                                        },
+                                        "AttackType": {
+                                            "items": {
+                                                "type": "string"
+                                            },
+                                            "type": "array"
+                                        },
+                                        "DisplayIdentifier": {
+                                            "items": {
+                                                "type": "string"
+                                            },
+                                            "type": "array"
+                                        }
+                                    },
+                                    "type": "object"
+                                }
+                            }
+                        }
+                    }
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "microsoftsentinel": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('MicrosoftSentinelConnectionName'))]",
+                                "connectionName": "[variables('MicrosoftSentinelConnectionName')]",
+                                "id": "[subscriptionResourceId('Microsoft.Web/locations/managedApis', resourceGroup().location,'azuresentinel')]",
+                                "connectionProperties": {
+                                    "authentication": {
+                                        "type": "ManagedServiceIdentity"
+                                    }
+                                }
+                            },
+                            "azuremonitorlogs": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('AzureMonitorLogsConnectionName'))]",
+                                "connectionName": "[variables('AzureMonitorLogsConnectionName')]",
+                                "id": "[subscriptionResourceId('Microsoft.Web/locations/managedApis', resourceGroup().location,'azuremonitorlogs')]"
```

Comment:
Please check if azuremonitorlogs can also be a "ManagedServiceIdentity"

Reply:
> azuremonitorlogs

Hi @manishkumar1991 ,
as far as I am aware its not supported? do you have a working example? I get the following error when trying to switch to managed identity

> {
    "status": "Failed",
    "error": {
        "code": "InvalidApiConnectionAlternativeParameters",
        "message": "The API connection 'azuremonitorlogs-EnrichDynatraceApplicationSecurityAttackIncident' has invalid inputs. Parameter value type cannot be set to 'Alternative' because managed identity is not supported in the API definition's connection parameters."
    }
}